### PR TITLE
import feed_publishers

### PR DESCRIPTION
**Pull request by xlqian**
import feed_publishers for stop_area

https://jira.canaltp.fr/browse/NAVP-717

### CODE REVIEW

**Comment by xlqian** [comment]
adding tests

File: src/bin/ntfs2mimir.rs [hunk 1]
```diff
@@ -82,6 +82,24 @@ fn to_mimir(
             name: navitia.physical_modes[pm_idx].name.clone(),
         })
         .collect();
+
+    let feed_publishers = navitia
+        .get_corresponding_from_idx(idx)
+        .into_iter()
+        .map(|contrib_idx| mimir::FeedPublisher {
+            id: format!("physical_mode:{}", navitia.contributors[contrib_idx].id),
```
Comment (TeXitoi) [inline src/bin/ntfs2mimir.rs]: I don't think physical_mode is the good solution...

File: src/bin/ntfs2mimir.rs [hunk 2]
```diff
@@ -82,6 +82,24 @@ fn to_mimir(
...
+    let feed_publishers = navitia
+        .get_corresponding_from_idx(idx)
+        .into_iter()
+        .map(|contrib_idx| mimir::FeedPublisher {
+            id: navitia.contributors[contrib_idx].id.clone(),
+            name: navitia.contributors[contrib_idx].name.clone(),
+            license: navitia.contributors[contrib_idx]
+                .license
+                .clone()
+                .unwrap_or(format!("")),
```
Comment (TeXitoi) [inline src/bin/ntfs2mimir.rs]:
```rust
.unwrap_or_else(|| "".into()),
```

File: src/bin/ntfs2mimir.rs [hunk 3]
```diff
@@ -82,6 +82,24 @@ fn to_mimir(
...
+            url: navitia.contributors[contrib_idx]
+                .website
+                .clone()
+                .unwrap_or(format!("")),
```
Comment (TeXitoi) [inline src/bin/ntfs2mimir.rs]: same

File: src/bin/ntfs2mimir.rs (hunk 2 above)
Comment (TeXitoi) [reply src/bin/ntfs2mimir.rs]: or `to_string()` if you prefer

File: tests/bragi_ntfs_test.rs [hunk 4]
```diff
@@ -105,6 +105,13 @@ fn gare_de_lyon(bragi: &BragiHandler) {
             {"key": "awesome_system", "value": "id:4242"},
         ])
     );
+    assert_eq!(
+        stop.get("feed_publishers").unwrap(),
+        &json!([
+            {"id": "TGC", "license": "DoWhattheFuckYouWantToPublicLicense",
```
Comment (TeXitoi) [inline tests/bragi_ntfs_test.rs]: :+1:

**Pull request merged**